cibuildwheel/__main__.py

```diff
@@ -259,6 +262,10 @@ def main() -> None:
     else:
         assert_never(platform)
 
+    if not identifiers:
+        print("ERROR: No build identifiers selected!")
```

Comment:
The current kind of errors are something like:

```py
print(f'cibuildwheel: Unsupported platform: {platform}', file=sys.stderr)
```

So I suggest following this convention, and going with:

```py
print(f'cibuildwheel: No build identifiers selected: {build_selector}', file=sys.stderr)
```